package.json

```diff
@@ -10,7 +10,8 @@
     "prettier": "prettier --write --ignore-unknown .",
     "prettier:check": "prettier --check --ignore-unknown .",
     "test": "yarn prettier:check && yarn lint",
-    "postinstall": "husky install"
+    "postinstall": "husky install",
+    "generate-translations": "extract-messages -l=en,es -o translations -d en --flat true **/messages.js"
```

Comment:
This command updates the translations files and should be run whenever a `messages.js` file it created, updated, or deleted.